### DISCUSSION TITLE
Zotero的翻譯和筆記功能，前後多出abc的符號貼。

### DISCUSSION THREAD

**Discussion by alicebear1203** [zotero-chinese/wiki#219] [求助 / Q&A] [answered]
<img width="702" alt="截圖 2024-06-28 下午6 24 56" src="https://github.com/zotero-chinese/wiki/assets/69949230/3427fc09-6457-4af2-86ba-a749952359ae">
如圖所示，Zotero的翻譯和筆記功能（不大確定怎麼形容），前後多出abc的符號貼。
之前Zotero6就有這樣的情況，以為升級會改善，看來是沒有...

**Comment by northword** [comment, marked as answer]
Translate 插件

![image](https://github.com/zotero-chinese/wiki/assets/44738481/9c1e4183-f102-44d7-8f48-0f335c389d51)

**Reply by alicebear1203** [reply]
@northword  太感謝了！自己看了半天不知道該怎麼辦，謝謝您！
